lib/sierra-models/base.js

```diff
@@ -136,7 +131,7 @@ class SierraBase {
 
   // returns array of objects. value is concatenation of contents of
   //   specified subfields - {value: 'val parallel link', subfieldMap: {a:'val', 6:'parallel link'}}
-  _prettifyRawVarFields (rawFields, subfields, opts) {
```

Comment:
The legacy choice of "subfields" here creates naming issues later, so proposing a name that conveys this is a config argument indicating "which subfields" to use

Reply:
great this was confusing when i was working on it

Reply:
maybe we can call it includedSubfields for symmetry with opts.excludedSubfields?